models/order-alternative/src/order-from-cart-draft/generators.ts

```diff
@@ -0,0 +1,25 @@
+import { ReferenceDraft } from '@commercetools-test-data/commons';
+import {
+  fake,
+  Generator,
+  oneOf,
+  sequence,
+} from '@commercetools-test-data/core';
+import { orderState, paymentState, shipmentState } from '../constants';
+import { TOrderFromCartDraftRest } from '../types';
+
+export const restGenerator = Generator<TOrderFromCartDraftRest>({
+  fields: {
+    cart: fake(() => ReferenceDraft.random().typeId('cart')),
+    version: sequence(),
+    orderNumber: fake((f) => String(f.number.int({ min: 100000 }))),
+    purchaseOrderNumber: fake((f) => String(f.number.int({ min: 100000 }))),
+    paymentState: oneOf(...Object.values(paymentState)),
+    orderState: oneOf(...Object.values(orderState)),
+    state: fake(() => ReferenceDraft.random().typeId('state')),
+    shipmentState: oneOf(...Object.values(shipmentState)),
+    custom: null,
+  },
+});
+
+export const graphqlGenerator = restGenerator;
```

Comment:
If drafts are always the same between REST and GraphQL, do we need to export both variations or could we have one?

Reply:
At this point, I'm not sure if they always are the same. If they are, I agree it would be easier to maintain just one, although perhaps a little more tricky to understand from the consumer side why we only have one draft model representation and two query model ones.

Reply:
> consumer side why we only have one draft model representation and two query model ones.

That's fair but easy to document and should outweigh the "cost" of exposing two - subjectively.